watergun_assassin/user_profile/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+
```

Comment:
Empty test files make me weep. Can we add some coverage here? Should be pretty similar to what we were testing for creating a user.